.editorconfig

```diff
@@ -0,0 +1,10 @@
+# http://editorconfig.org
```

Comment:
Nice information @gustavnikolaj, I wanted to try spacemacs for a while, I think this is the right moment ;) (and sorry for the late response, traveling delays).